Xamarin.Forms.Platform.Android/ViewRenderer.cs

```diff
@@ -92,10 +92,16 @@ protected override void Dispose(bool disposing)
 					_container = null;
 				}
 
-				if (Element != null && _focusChangeHandler != null)
+				if (Element != null)
 				{
-					Element.FocusChangeRequested -= _focusChangeHandler;
-					_focusChangeHandler = null;
+					Element.Behaviors?.Clear();
+					Element.Triggers?.Clear();
```

Comment:
> Could you please write a bit more info, what does "this breaks reuse of Element" means?
the next time the element will be associated to a renderer, it'll mis the Behaviors and Triggers.

> This is a serious memory leak, I'm confused why this PR was closed.
This is a Pull Request, the PR is wrong, so we close it. This is not a bug tracking system. Closing the PR does not make any statement about the validity or the seriousness of the bug, only about the proposed fix.